Dockerfile

```
@@ -179,25 +85,18 @@ RUN mkdir -p /upx && curl -sSL https://github.com/upx/upx/releases/download/v${U
 
 COPY --from=protoc_builder /out/ /out/
 COPY --from=go_builder /out/ /out/
-COPY --from=rust_builder /out/ /out/
-COPY --from=swift_builder /protoc-gen-swift /out/protoc-gen-swift
 RUN upx --lzma \
         /out/usr/bin/grpc_* \
         /out/usr/bin/protoc-gen-*
 RUN find /out -name "*.a" -delete -or -name "*.la" -delete
 
 FROM alpine:${ALPINE_VERSION}
-LABEL maintainer="Roman Volosatovs <roman@thethingsnetwork.org>"
+LABEL maintainer="Jaegertracing Authors"
```

We use this for Copyright headers, let's be consistent
```suggestion
LABEL maintainer="The Jaeger Authors"
```